app/collections/show.cjsx

```diff
@@ -0,0 +1,54 @@
+React = require 'react'
+PromiseToSetState = require '../lib/promise-to-set-state'
+talkClient = require '../api/talk'
+apiClient = require '../api/client'
+getSubjectLocation = require '../lib/get-subject-location'
+Paginator = require '../talk/lib/paginator'
+{Navigation} = require 'react-router'
+
+module?.exports = React.createClass
+  displayName: 'CollectionShow'
+  mixins: [Navigation, PromiseToSetState]
+
+  getInitialState: ->
+    subjects: []
+    subjectsMeta: {}
+    collection: {}
+
+  componentWillMount: ->
+    @setData(1) # start on page 1
+
+  subjectsRequest: (page) ->
+    collection_id = @props.params?.collection_id
+    apiClient.type('subjects').get({page, collection_id})
+
+  collectionRequest: ->
+    apiClient.type('collections').get({id: +@props.params?.collection_id}).index(0)
+
+  setData: (page) ->
+    @subjectsRequest(page).then (subjects) =>
+      @setState {subjectsMeta: subjects[0]?.getMeta()}, =>
+        @promiseToSetState {
+          collection: @collectionRequest()
+          subjects: @subjectsRequest(page)
+          }
+
+  goToPage: (n) ->
+    @transitionTo(@props.path, @props.params, {page: n})
+    @setData(n)
+
+  subject: (d, i) ->
+    # 'http://' may need to prefix some older subjects...
+    <img key={i} src={getSubjectLocation(d).src} />
```

Comment:
Yeah it will be--once that page is built I'll point any subject reference in collections/talk to it